### ISSUE TITLE
Unable to associate .svelte file with twigParser

### ISSUE THREAD

**Issue by jobelenus** [pgilad/leasot#114]
.svelte files (https://svelte.dev) are very similar to .vue files so I figured since the .vue used the twigParser I'd see if that works. But I am getting an error trying to run it

```
$ leasot -r markdown -t .svelte -A [.svelte, twigParser] frontend/*.svelte
/Users/john/.nvm/versions/node/v12.10.0/lib/node_modules/leasot/dist/lib/parsers.js:83
            throw new TypeError(`Cannot register extension: invalid extension ${extension}`);
```

Am I doing it wrong? Is there something preventing this from working? I took a few minutes and tried to find out where the file extensions were associated with parsers in the codebase, and I couldn't find it. I'd gladly make a PR to make this a "default" behavior.

Thanks for your project!

**Comment by pgilad** [comment]
Hey, I think the documentation on cli association should be improved. The correct usage will be:
`leasot -r markdown -t .svelte -A '.svelte,twigParser' frontend/*.svelte`

**Comment by jobelenus** [comment]
Aha! Well that runs... but finds no comments, which is sad. They are in the //, /* formats.

```
$ leasot -r markdown -t .svelte -A '.svelte,twigParser' frontend/*.svelte
### TODOs
| Filename | line # | TODO
|:------|:------:|:------
```

```
frontend/App.svelte:  //TODO figure out why a store wasn't working here
frontend/OwlOverview.svelte:  //TODO need to get this loaded via login
frontend/Owls.svelte:   * TODO Save the user pref whenever they click the toggle
```

**Comment by pgilad** [comment]
Yeah that's kind of weird to explain, but if you take a look [here](https://github.com/pgilad/leasot/blob/master/src/lib/parsers.ts#L64) the `.vue` files actually get associated not only with the `twigParser`, but with the parsers of `.html, .js, .css` file types. So one simple approach would be:
`leasot -A '.svelte,twigParser' -A '.svelte,defaultParser' 'frontend/*.svelte'`

**Comment by pgilad** [comment]
Feel free to add official support for `.svelte` 🙏

**Comment by jobelenus** [comment]
Where is that association? I couldn't find it

**Comment by pgilad** [comment]
https://github.com/pgilad/leasot/blob/master/src/lib/parsers.ts#L63

You need to add one for `.svelte`

**Comment by jobelenus** [comment]
Travis build in progress: https://github.com/pgilad/leasot/pull/115
